modules/oe_starter_content_featured_media_field/oe_starter_content_featured_media_field.info.yml

```diff
@@ -0,0 +1,13 @@
+name: OpenEuropa Starter Content Featured Media Field
+description: Contains the Starter Content Featured media field storage.
+package: OpenEuropa Starter Content
+
+type: module
+core_version_requirement: ^9.2
+dependencies:
+  - drupal:entity_browser
```

Comment:
Why this dependency?

Reply:
As per conversation it is not needed, also since we're adding oe_media it already has entity_browser as a requirement.